src/runtime/propensity-server.js

```diff
@@ -14,7 +14,34 @@
  * limitations under the License.
  */
 import {Xhr} from '../utils/xhr';
-import * as ServiceUrl from './services';
+import {adsUrl} from './services';
+import {Event} from '../api/propensity-api';
+import {AnalyticsEvent,EventOriginator} from '../proto/api_messages';
+import {isObject,isBoolean} from '../utils/types';
+import {ExperimentFlags} from './experiment-flags';
+import {isExperimentOn} from './experiments';
+
+/** @private @const {!Object<number,string>} */
```

Comment:
This is @private to which class? Should this be inside the constructor?

Reply:
It's a non-exported private const used for mapping different event types.  I could certainly put it inside the class if you think it should go there.  It's used exclusively by 1 line of code in propensity-server.